pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2088,6 +2088,22 @@ self: super:
   gi-gtk_4 = self.gi-gtk_4_0_12;
   gi-gdk_4 = self.gi-gdk_4_0_10;
 
+  # Missing dependency on gi-gdkpixbuf
+  # https://github.com/haskell-gi/haskell-gi/commit/889f478456b38425eca7df42b01f85fae626f113
+  # Fixed in 2.91.35
```

Comment:
Could you then maybe add an assert on the version number? Otherwise we are likely to forget this hack with possiblly weird results/error messages